### DISCUSSION TITLE
Add search functionality by actor/actress

### DISCUSSION THREAD

**Discussion by Treveo** [trakt/showly#895] [Ideas / Feature Requests]
**Description:**
Currently, to view all content featuring a specific actor or actress, I first need to search for a movie or show I know they are in, and then select the actor to see the rest of their filmography. It would be much more efficient to have the option to directly search by actor/actress.

**Use case:**
As a user, I want to be able to search for and view all shows and movies featuring a specific actor/actress, without needing to search for individual titles first. This would make it easier to explore an actor’s or actress’s entire filmography.

**Expected behavior:**
A search option is added where users can input the name of an actor or actress directly.
The app displays all relevant actors or actresses.

**Additional information:**
This feature would significantly improve the search experience, eliminating the need to first find a title and then select the actor. It would make discovering new content with favorite actors or actresses much faster and more intuitive.

**Comment by MidlajYH** [comment]
+1

This would save so much time. I would like to see this implemented in Showly.

**Comment by michaldrabik** [comment]
I'd actually like to have it myself already but focusing on something else right now :) But its on the map for sure.
